### ISSUE TITLE
Feature Request: Page Up / Page Down / Home / End

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#127]
```
Standard functionality users are familiar with from Browsers, Acrobat Reader, 
word processing, etc.
```

Original issue reported on code.google.com by `Anson....@gmail.com` on 15 Oct 2010 at 6:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@devaldi.com` on 20 Oct 2010 at 2:47
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
yes! this is essential !
```

Original comment by `zixzi...@gmail.com` on 9 Nov 2010 at 6:53

**Comment by GoogleCodeExporter** [comment]
```
Actually, it's already working for me in version 1.4.0, but I first have to 
click in the searchbox.
```

Original comment by `FL5...@gmail.com` on 23 Jan 2011 at 8:48

**Comment by GoogleCodeExporter** [comment]
```
Available Fixed in 1.4.2, FlexPaper has to be activated by clicking somewhere 
in FlexPaper for the keys to work though. 
```

Original comment by `erik.eng...@devaldi.com` on 11 Apr 2011 at 12:33
- Changed state: **Fixed**
